prow/config/jobs/istio.yaml

```diff
@@ -76,6 +76,24 @@ jobs:
       - name: TEST_SELECT
         value: "-multicluster"
 
+  - name: integ-telemetry-istiodless-mc-k8s-tests
+    modifiers:
+      - optional
+      - hidden
+      - skipped
+    command:
+      - entrypoint
+      - prow/integ-suite-kind.sh
+      - --topology
+      - MULTICLUSTER
+      - test.integration.telemetry.kube
+    requirements: [kind]
+    env:
+      - name: TEST_SELECT
+        value: "-multicluster"
```

Comment:
I thought we were trying to get rid of the multicluster tests? Should we run networking tests instead?

Reply:
This flag removes the multicluster label selector which targets the multicluster suite. The _last_ thing in that suite is the external istiod stuff, which we can hopefully put into our outer topology.

Reply:
The added jobs are dupes of `pilot-` `telemetry-` and `security-`